src/_posts/languages/ruby/rails/2000-01-01-start.md

```diff
@@ -120,6 +120,23 @@ $ git commit -m "Configure application for Scalingo"
 $ git push scalingo master
 ```
 
+### Database Migration
+
+To run a migration, you need to use a [post-deployment hook]({% post_url platform/app/2000-01-01-postdeploy-hook %}).
+
+The Ruby on Rails application starts up before the schema has changed, but does not receive a query. The way ActiveRecord works on Rails is that the schema is loaded into memory on the first query on a given table.
+
+As a result, if initializers, for example, or an asynchronous worker start performing operations before the migration has been executed, the application may load the schema `pre-migration`, and so a restart may be necessary after the migration, however short it may be.
```

Comment:
The part "if initializers, for example, or an asynchronous worker" is too long and makes the sentence complicated to read.

This is a really long sentence that I'm not sure to understand. I don't understand the "however short it may be" sentence.

Reply:
Ok, trying to make it clearer !